src/test/java/io/xstefank/wildfly/bot/utils/MockedContext.java

```diff
@@ -88,12 +88,8 @@ public MockedContext repoFiles(String... files) {
         return this;
     }
 
-    /**
-     * Sets the collaborators for the Pull Request and creates `GHUser`s for retrieval
-     * See GitHub#getUser()
-     */
-    public MockedContext collaborators(String... collaborators) {
-        this.collaborators.addAll(new HashSet<>(Arrays.asList(collaborators)));
+    public MockedContext users(String... users) {
+        this.users.addAll(new HashSet<>(Arrays.asList(users)));
```

Comment:
I'm not sure if creating 2 objects is better then just to simply iterate over the array

Reply:
I will remove the HashSet creation, I am not exactly sure, why I used it there